devito/operator.py

```diff
@@ -146,8 +146,6 @@ def __init__(self, nt, shape, dtype=np.float32, stencils=[],
                                      space_dims=self.space_dims,
                                      compiler=self.compiler, profile=profile,
                                      cache_blocking=cache_blocking, block_size=block_size)
-        self.propagator.time_loop_stencils_b = self.propagator.time_loop_stencils_b + getattr(self, "time_loop_stencils_pre", [])
-        self.propagator.time_loop_stencils_a = self.propagator.time_loop_stencils_a + getattr(self, "time_loop_stencils_post", [])
```

Comment:
I see it as you removing a shortcut in fact :) I put this here so that the application developer who is building an Operator sub-class does not need to be aware that something called a `Propagator` exists backstage. Why complicate their life with our internal representations when it can all be done from within `Operator`?